README.md

```diff
@@ -40,11 +40,11 @@ Data Factory does not support unit testing out of the box. The only way to valid
 
 ## Getting started
 
-1. Set up an empty Python project with your favorite testing library
-2. Install the package using your preferred package manager:
-   * Pip: `pip install data-factory-testing-framework`
-   * Poetry: `poetry add data-factory-testing-framework`
-3. Start writing tests
+### Start writting tests
+
+To get started using the tests, refer to the following [README](./examples/README.md)
+
+### Start contributing to the framework (PLACEHOLDER - TO BE UPDATED)
```

Comment:
We do have a section "Contributing" a bit further down. Is that where you would like to add additional contributing information?

Reply:
Yes, the "Contributing" section at the bottom is generic for open source/legal. What I would like to add in this section is how to launch the Project from a devcontainer for contributing for example, but more importantly - how to understand the structure of the Project and start contributing. If this is the first time you access the framework - how do you start? You had a great diagram (flow) of how the framework was working at the beginning - then we did a lot of changes - but I think something like that could help folks to start. And I think is important to explain that first the framework need to validate the sintax - how that works (or references to the lex behaviour) and then it need to reflect the functionality in a python implementation. So is a contributing more targeted to our use case.

Reply:
Got it, thanks for elaborating! 

Would you mind removing those placeholders from the readme and track them in our project / issue board? This keeps things clean and doesn't distract the reader.